ci-operator/step-registry/cucushift/installer/rehearse/azure/aks/hypershift/heterogeneous/provision/cucushift-installer-rehearse-azure-aks-hypershift-heterogeneous-provision-chain.yaml

```diff
@@ -44,5 +43,7 @@ chain:
     default: "2"
   - name: HYPERSHIFT_OPERATOR_PULL_SECRET
     default: "true"
+  - name: POD_SECURITY_ADMISSION_ANNOTATION
+    default: "false"
```

Comment:
This one does need POD_SECURITY_ADMISSION_ANNOTATION to be true imo

Reply:
Ill change it, I was just setting them wherever we were not retrieving the guest annotations before